app/controllers/api/v1/transactions/find_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::Transactions::FindController < ApplicationController
+  def show
+    render json: Transaction.find_by(merchant_params)
+  end
+private
+  def merchant_params
```

Comment:
Should this be transaction_params since we are finding transactions not merchant